test/gtest/gpu_mha_forward.cpp

```diff
@@ -76,58 +78,55 @@ struct TestCase
     float dropout;
 };
 
-inline std::vector<TestCase> GetSmokeTestCases()
+inline std::vector<TestCase> GetSmokeCases()
 {
-    if(CheckFloatArg("--float"))
-    {
-        return {
-            {9, 8, 8, 8, 0.0f},
-            {1, 2, 4, 5, 0.0f},
-            {2, 1, 5, 4, 0.0f},
-            {4, 2, 1, 3, 0.0f},
-            {5, 3, 4, 1, 0.0f},
-            {1, 2, 65, 5, 0.0f},
-            {2, 1, 67, 4, 0.0f},
-            {8, 7, 68, 1, 0.0f},
-            {1, 2, 257, 5, 0.0f},
-            {2, 1, 259, 4, 0.0f},
-            {8, 7, 270, 1, 0.0f},
-            {1, 1, 1, 1, 0.0f},
-            {3, 5, 32, 7, 0.8f},
-            {2, 2, 64, 128, 0.8f},
-            {2, 1, 128, 4, 0.8f},
-            {2, 7, 256, 31, 0.8f},
-        };
-    }
-    else
+    if(!(CheckFloatArg("--float") || CheckFloatArg("--float8")))
```

Comment:
Since we are still using a strange way to limit the tests, right now there is no way to turn on only fp8 tests.
Moreover having a dedicated build for just fp8 tests seems strange - why we should rebuild whole library just for it?

@atamazov @JehandadKhan @xinlipn what do you think?

Reply:
@CAHEK7 I would recommend implementing this feature just like we've implemented it for other data types. I believe we'll have more and more float8 tests in the future.